Kontent.Ai.Management.Tests/CodeSamples/CmApiV2.cs

```diff
@@ -2025,7 +2025,15 @@ public async void PutVariantWorkflow()
         var exception = await Record.ExceptionAsync(async () =>
                 await client.ChangeLanguageVariantWorkflowAsync(
                     new LanguageVariantIdentifier(itemIdentifier, languageIdentifier),
-                    new WorkflowStepIdentifier(Reference.ById(Guid.Empty), workflowStepIdentifier)
+                    new ChangeLanguageVariantWorkflowModel(Reference.ById(Guid.Empty), workflowStepIdentifier)
```

Comment:
Since this file is apparently used for code samples, I'd probably fix the formatting here.